src/utils/SourceMapManager.js

```diff
@@ -33,10 +32,7 @@ export class SourceMapManager {
     // Use presence of buck root argument to indicate which path format sourcemap prefixes take on.
     if (buckRoot !== undefined) {
       if (sourceMaps === undefined) {
-        throw new DebuggerError(
-          "Invalid input",
-          "Can't provide a sourcemap directory root without having sourcemaps present"
-        );
+        throw new Error("Invalid input: Can't provide a sourcemap directory root without having sourcemaps present");
```

Comment:
Why are you changing DebuggerError to Error?

Reply:
Initially because SourceMap was no longer specific to the debugger. However on further thought, this condition is not an error. It is explained in the comment:

>       // A buckRoot is unnecessary if there are no sourcemaps to translate between.
        // It actually causes paths to become incorrect, preventing proper files from opening.

Reply:
I do not understand your comment. How can `buckRoot` specified but not `sourceMaps`? It seems should be an error.

Reply:
`buckRoot` is just an CLI input. It can be disregarded if there are no sourceMaps. 

I'm saying there is a way to proceed if we are given a `buckRoot` but no sourceMap, and doing that does not necessarily mean an error will occur.